internal/api_server/agentserver/server.go

```diff
@@ -218,6 +218,7 @@ func (s *AgentServer) prepareHTTPHandler(serviceHandler service.Service) (http.H
 			s.orgResolver,
 			tlsmiddleware.CertOrgIDExtractor,
 		),
+		tlsmiddleware.UserAgentLogger(s.log),
```

Comment:
This will have performance implications if we are logging user-agent for all incoming messages from all devices.